Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM alpine
+
+RUN apk add --update --no-cache \
+    cmake \
+    make \
+    g++ \
+    git \
+  && rm -rf /var/cache/apk/*
+
+ADD ./ /MMDVMHost
+WORKDIR /MMDVMHost
+RUN make \
+&& cp MMDVMHost /usr/local/bin
```

Comment:
Not everyone will want their MMDVMHost binary in this location, I would suggest removing this line so it is left in situ once compile.  If folks want to move it, they can add the line themselves.

Reply:
In this case you must add an entrypoint file. Also, do not copy the docker-compose.yml without adapting it to its environment. For example, the path MMDVM.ini must be specified.